From f98239682cf366abd243276f73da5fcaba8ad589 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Thu, 13 Apr 2023 08:17:40 +0200
Subject: [PATCH] Also remote linker flags from ctk

For some reason, on zeus rpath creeps in although nothing is explicitcly
setting it
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5218437..314e1ea 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -133,8 +133,8 @@ set_target_properties(testmodule PROPERTIES PREFIX "")
 set_target_properties(${boost_python_core_module} PROPERTIES PREFIX "")
 
 # obey link flags from DeviceAccess (e.g. rpath)
-set_target_properties(testmodule PROPERTIES LINK_FLAGS "${ChimeraTK-DeviceAccess_LINKER_FLAGS}")
-set_target_properties(${boost_python_core_module} PROPERTIES LINK_FLAGS "${ChimeraTK-DeviceAccess_LINKER_FLAGS}")
+#set_target_properties(testmodule PROPERTIES LINK_FLAGS "${ChimeraTK-DeviceAccess_LINKER_FLAGS}")
+#set_target_properties(${boost_python_core_module} PROPERTIES LINK_FLAGS "${ChimeraTK-DeviceAccess_LINKER_FLAGS}")
 
 # Copy the pure python module to the build directory. This is needed by the
 # tests. Packaging looks for it in this location. The version information in the
